### DISCUSSION TITLE
Playtime: Add 'conditional looping' functionality

### DISCUSSION THREAD

**Discussion by AudioBabbL** [helgoboss/helgobox#1049] [Ideas]
Please can we have 'conditional looping' where a clip can have an offset start point for looping that is different to the start point for first launch, and also a 'tail' that plays after the clip is stopped. The following illustrates it:

![playtime-2_looping](https://github.com/user-attachments/assets/321fcb0f-f0f4-49d3-aca6-77bb2a58b818)

I could be wrong, but I think this functionality is available in Ableton / Bitwig.

**Comment by helgoboss** [comment]
Pretty sure it's available in Ableton/Bitwig. That fact alone doesn't automatically make it a Playtime must-have, but I strongly consider adding this at some point. It's too easy to not do it ;)

Will have to wait for the time after final release though!

**Comment by AudioBabbL** [comment]
Might have been a bit 'lazy' of me to just quote Abelton/Bitwig as a justification!

Two scenarios where this feature would be useful:

1. say you have a bass loop that has a short fill or run before it comes in. Or, maybe the bass loop starts on a high note to come in, plays the loop for one bar, but once it's looping you want it to have a low note on the downbeat (so you want it to loop bar 2, which starts on a low note). Then... at the other end, when you stop the loop you want a note on the downbeat of the next bar (so the loop doesn't stop 'abruptly).

2. Reverbs/Delays:  If you have a loop with reverb or delay in it, then the first time when you hit play, you want it to start 'dry' and the reverbs/delays build up. But when it's looping you want to hear the tails from the end of the loop. Then, at the end, when you hit stop, you don't want the reverbs/delays to cut off abruptly, so you need to hear a tail after the end of the loop.
